terraform/variable.tf

```diff
@@ -0,0 +1,90 @@
+variable "cloud_storage_credentials" {
```

Comment:
new to Terraform, show me some day ... :)

Reply:
certainly :) it's pretty nifty